README.md

```diff
@@ -42,25 +42,53 @@ pip install matcha-ml
 matcha provision
 ```
 
-After the provisioning completes, run the following bash script
+After the provisioning completes, we will have on hand these resources:
+* Kubernetes cluster on Azure
+* Seldon Core installed on this cluster
+* Istio ingress installed on this cluster
```

Comment:
Question: Do we need a ingress for seldon? Can streamlit not access seldon service in same way to chroma server and we just add one ingress on top of streamlit?

Reply:
Yeah, we can talk to Seldon on k8s directly. I wouldn't disable ingress until streamlit app is deployed to the cluster